common/lib/xmodule/xmodule/modulestore/tests/utils.py

```diff
@@ -194,7 +194,7 @@ class MemoryCache(object):
     the modulestore, and stores the data in a dictionary in memory.
     """
     def __init__(self):
-        self._data = {}
+        self.data = {}
```

Comment:
It's because Split Modulestore expects the given request_cache object to store its `dict` in a `data` field.  For example: https://github.com/edx/edx-platform/blob/master/common/lib/xmodule/xmodule/modulestore/split_mongo/split.py#L792